### ISSUE TITLE
Sometimes when I open or close files - it get stucked and sometimes I even have to restart Atom

### ISSUE THREAD

**Issue by MartinIPStudios** [h3imdall/ftp-remote-edit#448]
Basically - when I open project, browse trough the files. 

Sometimes when I save, it get stuck. Sometimes when I open, it get stuck. I always see this error text:

**Comment by pitva90** [comment]
I am using the latest version 0.20.0 and I have the same problem. But for me - after about 10 minutes the file uploads successfully. It was happening in Atom and now it is the same in Pulsar.

**Comment by MartinIPStudios** [comment]
Yes, this problem is in both - Atom and Pulsar. When you grind and write code a lot - this thing happens really often and get on nerves. Sadly I have completely no clue where to find what is causes. This kinda gives a lot of information - 

User launched a task while another one is still running. Forgot to use 'await' or '.then()'?

But I have no clue where to find it :/

**Comment by Morpheus211** [comment]
This also happens to me and is indeed very annoying. According to my hoster, the maximum number of simultaneous connections to the ftp server is exceeded. When I close Pulsar, some or all connections are closed. Then I can connect again.

Maybe this will help to solve the problem?
